### PR TITLE
Update 2022 - Legiones Astartes.cat

### PR DESCRIPTION
**Pull request by Toreador13**
Cleaned up and completed Assault and Despoiler squads. Fixed Toxin Bombs and Dark Channeling so they would work.

Added some more groups entries for Achean Force weapons

### CODE REVIEW

**Comment by Toreador13** [comment]
Ugh, not sure why this one is so messy. I cleaned up the Sgts and made their options flatter, so that is why so many deletes, not sure what else is going on there. Did a lot of cleanup and added all the legion special options..

**Comment by nstephenh** [comment]
This one's messy because you don't have my changes. Try merging in from master first.

**Comment by Toreador13** [comment]
I'll try that later tonight. Thought I had them when I started. I'll give it another go.

**Review by nstephenh — CHANGES_REQUESTED** [review]
Looks like you've completely deleted the Legiones Astartes file here...

EDIT: These were my review marks from my first review, but now the file appears completely gone.

File: 2022 - Legiones Astartes.cat [hunk 1]
```diff
@@ -21024,6 +21786,308 @@ A model with this special rule, and any unit it joins, must either begin any bat
         </entryLink>
       </entryLinks>
     </selectionEntryGroup>
+<<<<<<< Updated upstream
```
[the remark this answers is not in the document]
Comment (Toreador13) [reply 2022 - Legiones Astartes.cat]: I may just start over again. I looked at that also and it seemed weird, and when I opened the file It seemed to have my changes. I think I may close this, and start over. Maybe just get master again and make my changes again.

**Comment by Toreador13** [comment]
Closing because something is entirely wrong with this.

**Pull request closed without merging**